filters/auth/oidc.go

```diff
@@ -242,19 +243,29 @@ func (s *tokenOidcSpec) CreateFilter(args []interface{}) (filters.Filter, error)
 		validity = defaultCookieValidity
 	}
 
+	oidcClientId := sargs[paramClientID]
+	if oidcClientId == "" {
+		oidcClientId, _ = os.LookupEnv("OIDC_CLIENT_ID")
```

Comment:
we should lookup once in skipper.go and pass the result values via `OidcOptions`

Reply:
see also https://github.com/zalando/skipper/blob/master/skipper.go#L1674-L1680

Reply:
The existing OidcOptions struct does not have OIDC_Client_ID - should that be added to the field as a struct?
In addition would the arguments parsing in oidc.go have to be updated? https://github.com/zalando/skipper/blob/master/filters/auth/oidc.go#L80